ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install.sh

```diff
@@ -29,10 +32,10 @@ $UBUNTU_COMMON_DIR/install_nccl.sh
 $UBUNTU_COMMON_DIR/install_docker.sh
 
 # cleanup downloaded tarballs - clear some space
-rm -rf *.tgz *.bz2 *.tbz *.tar.gz *.run *.deb *_offline.sh
-rm -rf /tmp/MLNX_OFED_LINUX* /tmp/*conf*
-rm -rf /var/intel/ /var/cache/*
-rm -Rf -- */
+#rm -rf *.tgz *.bz2 *.tbz *.tar.gz *.run *.deb *_offline.sh
```

Comment:
Once the current test build runs and is verified, these lines to cleanup install will be uncommented